packages/code-du-travail-frontend/cypress/support/before.ts

```diff
@@ -0,0 +1,50 @@
+// @ts-ignore
+import fs from "fs";
+
+export const downloadAllUrlsToValidate = async () => {
+  const urls: string[] = ["/convention-collective", "/contribution"];
+
+  const response = await fetch(
+    "https://code-du-travail-numerique-preprod.ovh.fabrique.social.gouv.fr/api/plan-du-site"
```

Comment:
ici si je mets `${SITE_URL}/api/plan-du-site` j'ai une erreur parce que cette variable n'est pas rensigné

Reply:
Essaye : `process.env.TEST_BASEURL`

Reply:
j'ai cette erreur sur `process` :
`TS2591: Cannot find name process. Do you need to install type definitions for node? Try npm i --save-dev @types/node and then add node to the types field in your tsconfig.`